### PR TITLE
[DONE] Transpiling files to ES6 and more

### PR DESCRIPTION
**Pull request by gkal19**
Transpiling files to ES6, Indents and minor fixes

### CODE REVIEW

**Comment by gkal19** [comment]
Todos os erros apontados no Travis foram corrigidos.
Está tudo pronto! :+1:  :rocket: :

**Comment by raphamorim** [comment]
Ok, let's review. Thanks for contributing, I've some notes:
1- `webpack.config.js` and `main.js` does not change to ES2015+, because they run on node_environment, unless we set node version on a `nvm` file separatelly.
2- Curiosity: why change every function to arrow function? Some functions there need run in a context scope. Seems to work, but I only have to understanding this change.

**Comment by gkal19** [comment]
@raphamorim, First of all I would like to say that I am a fan of yours and your projects.

The changes to the original code in `webpack.config.js` and` main.js` have already been made.

Now, talking about the `arrow function`. It can be used to define anonymous functions. Of course, it has a shorter syntax when compared to function expressions and links the value of `this` lexically.

In short: `Arrow Functions` can make your JavaScript code much more beautiful and solve any problems with` this`.

> In the global execution context (outside of any function), this refers to the global object, whether in strict mode or not.

I hope you understood.

Note: There are some times when you should not use the arrow functions. So I recommend you read this [article](https://udgwebdev.com/quando-nao-usar-arrow-functions/).

**Comment by raphamorim** [comment]
Nice, I'll test it tonight and reply here.

About arrow function I'm just being just implicant te-hehe, doesn't matter with it.

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
 dist
-node_modules
 ~/
-*.log
\ No newline at end of file
+
+# Logs
+logs
+*.log
+npm-debug.log*
```
Comment (raphamorim) [inline .gitignore]: *.log already work for all log cases

**Comment by raphamorim** [comment]
Hi @gkal19 can you squash the commits?

**Comment by raphamorim** [comment]
Squash and merged in #34
Thanks for contributing :)

**Comment by gkal19** [comment]
@raphamorim It's me who should thank

**Pull request closed without merging**